### DISCUSSION TITLE
Proplems with running to code

### DISCUSSION THREAD

**Discussion by Quelt1c** [mtxr/vscode-sqltools#1305] [Q&A]
After installing sqltools, I don't have a button above the code to run active connection

**Comment by gjsjohnmurray** [comment]
Did you configure a connection and create a text file with a .sql extension.

Maybe paste screenshots here?

**Comment by gjsjohnmurray** [comment]
Closing due to lack of response from @Quelt1c. Can be reopened if that changes.
